third_party/move/move-vm/runtime/src/loader.rs

```diff
@@ -745,11 +742,128 @@ impl Loader {
             .collect::<PartialVMResult<Vec<_>>>()
             .map_err(|err| err.finish(Location::Undefined))?;
 
-        // verify type arguments
+        Ok((module, func, parameters, return_))
+    }
+
+    // Matches the actual returned type to the expected type, binding any type args to the
+    // necessary type as stored in the map. Returns true if a successful match is made.
+    fn match_return_type<'a>(
+        returned: &Type,
+        expected: &'a Type,
```

Comment:
Should probably mention that the expected type must be a concrete type, or in other words, does not contain any type parameters. Otherwise, it would require the full unification algorithm.

Reply:
Done